tsconfig.json

```diff
@@ -18,6 +18,14 @@
     "forceConsistentCasingInFileNames": true,
 
     "outDir": "./dist",
-    "rootDir": "."
+    "rootDir": ".",
   },
+  "exclude": [
+    "node_modules",
+    // Don't try to compile the already compiled stuff
+    "./dist",
+    // Don't do anything with test files as they will be compiled by
+    // ts-jest
+    "**/*spec.ts",
```

Comment:
Before adding this, if I had run a build/prepare and had some old test code in `/dist/__tests__/*spec.js`, then running `yarn test` would run both the latest tests in `/__tests__/` as well as the outdated tests in `/dist/__tests__/`...which could cause a failure that is hard to notice why and was driving me bonkers.